pom.xml

```diff
@@ -74,6 +77,62 @@
       <version>${sonar.version}</version>
```

Comment:
Release candidate of 5.2 is available in Maven central repository. GroupId is now `org.sonarsource.sonarqube` and version is `5.2-RC2`. This dependency should be fixed. Thanks.